src/pandapipes/io/convert_format.py

```diff
@@ -45,6 +45,9 @@ def _rename_columns(net):
             else:
                 net['controller'].drop('controller', inplace=True, axis=1)
         net["controller"].rename(columns={"controller": "object"}, inplace=True)
+    if "pipe" in net:
+        if "u_w_per_m2k" not in net["pipe"]:
```

Comment:
I think there are some caveats here wrt. where pandas looks for u. Rather do:

    if "u_w_per_m2k" not in net["pipe"].columns